app/src/main/java/com/github/piasy/biv/example/RecyclerViewActivity.java

```diff
@@ -18,7 +19,7 @@
     protected void onCreate(@Nullable Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
 
-        BigImageViewer.initialize(FrescoImageLoader.with(getApplicationContext()));
+        BigImageViewer.initialize(GlideCustomImageLoader.with(getApplicationContext(), SampleCustomImageSizeModel.class));
```

Comment:
This recycler view demo load image from wikipedia, does it support image size query? I tried it in my chrome browser, it doesn't work.

Reply:
Got it :)